.takeshape/pattern/schema.json

```diff
@@ -1,806 +1,1107 @@
-          },
-          {
-            "id": "prices",
-            "name": "rest:get",
-            "service": "stripe",
-            "path": "/v1/prices",
-            "searchParams": { "ops": [{ "path": "active", "value": "true" }] }
+        "description": "Get my subscriptions"
+      },
+      "getMyPayments": {
+        "shape": "Shopify_OrderConnection",
+        "resolver": {
+          "id": "getCustomerOrders",
+          "name": "graphql:query",
+          "service": "shopify-admin",
+          "fieldName": "orders",
+          "args": {
+            "ops": [
+              {
+                "path": "query",
+                "mapping": [
+                  ["get", {"path": "$claims.email"}],
+                  ["prepend", {"text": "email:"}]
+                ]
+              },
+              {"path": "first", "mapping": "$args.first"}
+            ]
           }
-        ]
+        },
+        "args": {"type": "object", "properties": {"first": {"type": "number"}}}
       },
-      "description": "Get Stripe products."
-    },
-    "getProfile": {
-      "shape": "Profile",
-      "resolver": {
-        "name": "takeshape:get",
-        "service": "takeshape:local",
-        "shapeName": "Profile"
-      },
-      "description": "Get a profile by ID",
-      "args": "TSGetArgs<Profile>"
-    },
-    "getProfileList": {
-      "shape": "PaginatedList<Profile>",
-      "resolver": {
-        "name": "takeshape:list",
-        "service": "takeshape:local",
-        "shapeName": "Profile"
-      },
-      "description": "Returns a list of profiles in natural order.",
-      "args": "TSListArgs<Profile>"
-    }
-  },
-  "mutations": {
-    "upsertMyProfile": {
-      "shape": "Profile",
-      "resolver": {
-        "results": {
-          "ops": [
-            { "path": "$", "mapping": "$resolvers.createProfile.result" },
-            { "path": "$", "mapping": "$resolvers.updateProfile.result" }
+      "getShopifyAndRechargeProducts": {
+        "shape": "ShopifyAndRechargeProducts",
+        "resolver": {
+          "results": {
+            "ops": [
+              {
+                "path": "$.recharge",
+                "mapping": [["get", {"path": "$resolvers.recharge"}]]
+              },
+              {
+                "path": "$.shopify",
+                "mapping": [["get", {"path": "$resolvers.shopify"}]]
+              }
+            ]
+          },
+          "compose": [
+            {
+              "id": "shopify",
+              "name": "graphql:query",
+              "service": "shopify-admin",
+              "fieldName": "products",
+              "options": {
+                "selectionSet": "{edges{node{id title description variants(first:3){edges{node{id inventoryQuantity}}} images(first:1){ edges{ node{ url } } } priceRangeV2{ maxVariantPrice{ amount currencyCode } }  sellingPlanGroupCount sellingPlanGroups(first:1){ edges{ node{ id } } } } }}"
```

Comment:
Because I need `Shopify_Product` defined in my schema for my `indexedShapes`, this creates a redundant situation. I have a `MyShopifyProduct`, which I created to match what's returned by this query, then I have `Shopify_Product` both as shapes in my schema. I wanna try indexing data with the `MyShopifyProduct` shape, but I'm getting a bug where I can't deploy my project atm so will try that later. If that works, I can delete `Shopify_Product`